### ISSUE TITLE
MultiSelect behaves unexpectedly when using TalkBack

### ISSUE THREAD

**Issue by georgi-sla** [telerik/kendo-angular#4632]
### Description

Currently, when the TalkBack accessibility tool is enabled and the user tries using the MultiSelect, the component's popup does not open as expected, selected items cannot be removed, and even if the items' popup is opened, the user cannot select them.

### Steps To Reproduce

1. Create a MultiSelect component.
2. Enable TalkBack.
3. Try opening the MultiSelect popup and selecting items.

### Screenshots or video

https://github.com/user-attachments/assets/2342fbd7-3a88-408e-8ea7-af7a6051101d

### Actual Behavior

In such a setup with TalkBack enabled, the user cannot open the items' popup of the MultiSelect, and it seems like the internal input is being focused. Furthermore, if the user taps on a selected item, the items' popup may sometimes open, but the user cannot remove the item. In case the items' popup gets opened, the user cannot select an item.

### Expected Behavior

The MultiSelect should behave as usual when the TalkBack accessibility tool is enabled.

### Browser

Firefox

### Browser version

latest

### OS type

Android

### OS version

Android 15

**Comment by georgi-sla** [comment]
Upon further internal discussions, we've determined that the current behavior is outside of our control and is, in fact, dependent on the specific speech-recognition tool being used.

In general, the Kendo UI for Angular components rely heavily on the DOM and the standard JavaScript APIs provided by supported browsers. These components are designed to respond to browser-native events such as `click`, `focus`, and `blur`, which are typically triggered through direct user interaction (e.g., keyboard or mouse).

On the other hand, OS-level speech-recognition technologies—such as Dictation on macOS, Siri and Voice Control on iOS, Cortana on Windows, Android Speech, TalkBack, and others—are primarily designed for native applications and may not trigger the same browser events expected by Kendo components. As a result, if a given speech-recognition tool does not emit these standard events, the components may not behave as expected.

With that being said, due to this reliance on browser-level event handling, we’re unable to provide built-in support or guaranteed compatibility with such speech-recognition technologies.
